tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1360,4 +1361,40 @@ public function commentsProvider() : array
             'valid with extra opening brackets' => ['(DC2Type:should((.stop)).before)', 'should((.stop', $currentType],
         ];
     }
+
+    public function testCreateAndListSequences() : void
+    {
+        if ( ! $this->_sm->getDatabasePlatform()->supportsSequences()) {
+            $this->markTestSkipped('This test is only supported on platforms that support sequences.');
+        }
+
+        $sequence1Name           = 'sequence_1';
+        $sequence1AllocationSize = 1;
+        $sequence1InitialValue   = 2;
+        $sequence2Name           = 'sequence_2';
+        $sequence2AllocationSize = 3;
+        $sequence2InitialValue   = 4;
+        $sequence1               = new Sequence($sequence1Name, $sequence1AllocationSize, $sequence1InitialValue);
+        $sequence2               = new Sequence($sequence2Name, $sequence2AllocationSize, $sequence2InitialValue);
+
+        $this->_sm->createSequence($sequence1);
+        $this->_sm->createSequence($sequence2);
+
+        /** @var Sequence[] $actualSequences */
+        $actualSequences = [];
+        foreach ($this->_sm->listSequences() as $sequence) {
+            $actualSequences[$sequence->getName()] = $sequence;
+        }
+
+        $actualSequence1 = $actualSequences[$sequence1Name];
+        $actualSequence2 = $actualSequences[$sequence2Name];
+
+        $this->assertSame($sequence1Name, $actualSequence1->getName());
```

Comment:
Please call all asserts statically.

Reply:
Isn't that deprecated?

Reply:
What do you mean? All assert methods are declared static: https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Assert.php

Reply:
Ah, not deprecated but it's there because of something oldhttps://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-148407321. 

Both can be used tho. I'm used to call it via non-static way. 

I guess it needs to be reworked to static then as the whole project uses static calls. Shall I?

Reply:
And bad design in PHPUnit apparently, since non-static TestCase extends static Assert class. :/
Since it's declared as static, it was decided to use static calls in tests for assert methods, otherwise it's different context (and personally I think PHP should emit a warning, just like static->instance call).